### DISCUSSION TITLE
Порядок вывода табов в цикле

### DISCUSSION THREAD

**Discussion by gorbarov** [totumonline/totum-ru-issues-and-discussions#413] [Как это сделать?] [answered]
Два небольших вопроса про табы в циклах
1) Можно ли сделать так, чтобы кнопка, которая ведёт на префильтр по другой таблице tab_NameTable подчинялась порядку сортировки? Сейчас они всегда раньше таблиц в цикле отображаются в табах

1.1) Можно ли сделать кнопку видимой, я так понимаю кнопка в цикл с префиксом tab_ в имени хардкодно скрывается?

2) Как поменять порядок отображения для обычных табов?

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/388620/3b3d0c8f-5124-4231-810c-6a021d8d3809)

**Comment by erpic-ru** [comment, marked as answer]
1. Поменять сорт в таблице циклов поставить 2000, например и таб будет в конце.
2. Меняете сорт в настройках расчетных таблиц в цикле, в новых циклах будет новый порядок. Для уже созданных нужно менять порядок в версиях

**Reply by mihail-cornev** [reply]
А если сортировка табов не работает, то что может быть причиной?

**Comment by DmitriyGulevichS** [comment]
> А если сортировка табов не работает, то что может быть причиной?

Актуальный вопрос. Выставляю сортировку в таблицах внутри цикла, но порядок не меняется ни в табах ни с списке таблиц (левое меню) 

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/100049818/730e5712-3c83-4438-a07d-2f0cecacebd9)

вот таблица с сортировкой 1000 

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/100049818/840b3fb7-1309-4d47-845d-c1726d2f8b6f)

Вот таблица с сортировкой 920 (я пробовал разные значения, от 1000 тоже)

![image](https://github.com/totumonline/totum-ru-issues-and-discussions/assets/100049818/6c44e69a-b673-407c-a8d2-c11f49bf8748)

**Reply by DmitriyGulevichS** [reply]
Нашел ответ тут https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/198
Вопрос не актуален
